source/search_page/assets/searchRecipe.js

```diff
@@ -79,21 +92,21 @@ class SearchRecipe extends HTMLElement {
         const timeNumb = document.createElement('p')
         timeNumb.setAttribute('class', 'recipe-time');
 
-        timeNumb.innerHTML = "14";
+        timeNumb.innerHTML = data.readyInMinutes + "\n min";
 
 
-        recipeImg.setAttribute('src','images/pasta.jpeg');
+        recipeImg.setAttribute('src',data.image);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8435053)
```suggestion
        recipeImg.setAttribute("src",data.image);
```